var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -358,11 +359,6 @@ def flag_handler(self, name, flags):
 
         return (flags, None, None)
 
-    def setup_build_environment(self, env):
-        # mvapich2 configure fails when F90 and F90FLAGS are set
```

Comment:
This should also probably be kept.

Reply:
I don't think so, since this is true for `mpich` and afaik `mvapich2` is an `mpich` derivative. It is not something special about `mvapich`.

Reply:
Oh I missed this hiding in the base class, I thought this was specific to something that `mvapich` added and was accidentally removed.